### ISSUE TITLE
Instructions for installing Hydraplay

### ISSUE THREAD

**Issue by Craftyalchemist** [mariolukas/HydraPlay#18]
Hi,

In the readme, the installation instructions for the Hydraplay are a bit vague. Could you please expand this section so it's clearer to understand.

Thanks

**Comment by mariolukas** [comment]
You are right. The setup is getting complex. I will try to build a docker solution as soon as i can.

**Comment by Craftyalchemist** [comment]
Hi,
Thanks for the quick reply.
The part I'm having trouble with is the following:

One more thing. You need to build HydraPlay before you place the files into /home/pi/hydraplay
1. Install node on the pi or on your local machine
2. change to the HydraPlay Source folder.
3. Install dependencies with npm install
4. change the ip and ports in environment.ts
5.Build with ng build
6.the result will be in target/build
7.copy all files in target/build to /home/pi/hydraplay on the pi
8. try to connect.

So,
1. I installed node.js 
2. created a folder at /home/crafty/hydraplay
3. In home/crafty/hydraplay I ran git clone https://github.com/mariolukas/HydraPlay.git.
4. I then ran npm install in home/crafty/hydraplay/HydraPlay

The issue I have is if I run ng build then a window opens up but nothing happens
I ran npm build-script which appeared to then use ng build which created a folder called dist/multiroom-gui.
I moved the files into the HydraPlay folder.

When I go to http://localhost/8800, I have a black screen.

Anyhelp, where I have gone wrong would be much appreciated.

Thanks in advance

**Comment by mariolukas** [comment]
There is a setup script now.
https://github.com/mariolukas/HydraPlay/tree/master/hydracontrol

**Comment by zionsuperfly** [comment]
hello all, 
For the moment i cannot get the build.
these steps i did:
1) I installed node.js
2) created a folder at /home/pi/hydraplay
3) In home/pi/hydraplay I ran git clone https://github.com/mariolukas/HydraPlay.git.
4) I then ran npm install in home/pi/hydraplay/HydraPlay/hydraplay

if i run npm install in home/pi/hydraplay/HydraPlay
npm WARN saveError ENOENT: no such file or directory, open '/home/pi/hydraplay/HydraPlay/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/home/pi/hydraplay/HydraPlay/package.json'
npm WARN HydraPlay No description
npm WARN HydraPlay No repository field.
npm WARN HydraPlay No README data
npm WARN HydraPlay No license field.

up to date in 0.582s
found 0 vulnerabilities


if i run in npm install in home/pi/hydraplay/HydraPlay/hydraplay,
it start to build something but with some warning, also no build file is visible


Installed to /home/pi/hydraplay/HydraPlay/hydraplay/node_modules/node-sass/vendor/linux-arm-79/binding.node
npm WARN @angular/platform-browser-dynamic@7.2.15 requires a peer of @angular/compiler@7.2.15 but none is installed. You must install peer dependencies yourself.
npm WARN @ngtools/webpack@7.3.9 requires a peer of @angular/compiler-cli@>=5.0.0 <8.0.0 || ^7.0.0-beta.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ngtools/webpack@7.3.9 requires a peer of typescript@>=2.4.0 < 3.3 but none is installed. You must install peer dependencies yourself.
npm WARN codelyzer@4.5.0 requires a peer of @angular/compiler@>=2.3.1 <8.0.0 || >7.0.0-beta <8.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm"})

added 75 packages from 62 contributors and audited 43725 packages in 694.879s

5 packages are looking for funding
  run `npm fund` for details

found 6 vulnerabilities (1 low, 3 moderate, 2 high)
  run `npm audit fix` to fix them, or `npm audit` for details


can somebody help me with this? what do i do wrong?

kind regards
